### PR TITLE
accounts: use slice instead of vec

### PR DESCRIPTION
**Pull request by cavemanloverboy**
addresses #2

### CODE REVIEW

File: harness/src/lib.rs [hunk 1]
```diff
@@ -177,8 +177,9 @@ impl Mollusk {
             .collect::<Vec<_>>();
 
         let transaction_accounts = [(self.program_id, self.program_account.clone())]
-            .into_iter()
+            .iter()
             .chain(accounts)
+            .cloned()
```
Comment (buffalojoec) [inline harness/src/lib.rs:182]: Yeah, this was exactly what I had. Unfortunate, but I agree the API is better served with a reference.

**Review by buffalojoec — COMMENTED** [review]
Lgtm, thanks! Just need that one touch-up to pass clippy (I think I broke the other one).

Since you've added the lifetime specifier to `Bench`, in a follow-up PR I'm going to add some more references there.

**Comment by cavemanloverboy** [comment]
fixed clippy

**Review by buffalojoec — APPROVED** [review]
Thanks!

**Pull request merged**